test/integration/404-page-ssg/pages/404.js

```diff
@@ -0,0 +1,4 @@
+export const getStaticProps = () => ({ props: { hello: 'world' } })
+
+const page = () => `custom 404 page ${Math.random()}`
```

Comment:
This will be a hydration error, plz move random value to `getStaticProps` return.

Reply:
Updated